ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-commands.sh

```diff
@@ -31,10 +31,17 @@ test_list_file="test-list"
 test_skips_file="test-skips"
 test_env_file="test-env"
 
+if [ ${TEST_TYPE:-""} == "minimal" ]; then
+    echo "Running minimal test list"
+    read -d '#' TEST_LIST << EOF
+[sig-cli] Kubectl client Kubectl api-versions should check if v1 is in available api versions
```

Comment:
```suggestion
"[sig-cli] Kubectl client Kubectl api-versions should check if v1 is in available api versions [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```
let's put it as a variable in the the workflow. Also, this string won't match any tests.

If it's a variable, I wonder if we can set `TEST_LIST: ${TEST_LIST_MINIMAL}` in job definition, so we won't have to define a new test type.

Reply:
I guess we can, I will try